### PR TITLE
#9 login form patch

### LINKED ISSUE #9: login form shouldn't validate password length
## Actual behaviour
I've successfully created a Speckle account with the API :
```js
{
  "email": "radug@speckle.works",
  "password": "testing",
  "name": "Radu",
  "surname": "Gidei",
  "company": "Grimshaw",
}
```
 but if i try to log in to Speckle server using the Admin app, it validates the password field, requiring it to be longer than 8 characters.

speckle login

## Expected behaviour
I should be able to log in using my pre-existing password.
Whilst validation is expected on `registration` page, it shouldn't be on `login` page.

### PR DESCRIPTION
**Pull request by radumg**
fixes #9 

cc @didimitrie

### CODE REVIEW

**Comment by mswaidan** [comment]
See comments on the original issue, but we also want to keep URL at 0.0.0.0. This allows dev servers to be accessed from addresses not on the LAN. E.g. I run my dev server on Digital Ocean rather than on localhost.

**Comment by didimitrie** [comment]
plus `package-lock.json` is a godsend, it needs to stay!

File: src/components/LoginForm.vue [hunk 1]
```diff
@@ -27,7 +27,7 @@
                 </md-field>
                 <md-field :class="{'md-invalid': errors.has('password')}">
                   <label>Password</label>
-                  <md-input v-model="password" type="password" v-validate name="password" data-vv-rules="min:8|required" data-vv-name="password" autocomplete="password"></md-input>
+                  <md-input v-model="password" type="password" v-validate name="password" data-vv-name="password" autocomplete="password"></md-input>
                   <span v-show="errors.has('password')" class="md-error">{{ errors.first('password') }}</span>
```
Comment (didimitrie) [inline src/components/LoginForm.vue]: also all the validate directives plus span below need to removed if we don't validate the passwd lenght

Comment (radumg) [reply src/components/LoginForm.vue]: 👍

**Comment by radumg** [comment]
ok, reverted the `localhost` change, but i think a more elegant solution is needed, the experience whilst developing `SpeckleAdmin` locally with the server remotely is currently a bit broken, `0.0.0.0` is launched.
Also see #11 , hot reloading wasn't working.

also reverted `package-lock.json` - if you're happy to have this constantly updated when we run `npm install` 👍

**Comment by radumg** [comment]
btw, thanks for the uber-quick replies @didimitrie & @mswaidan 🍻

**Comment by radumg** [comment]
Should be ok to merge now ?

**Comment by radumg** [comment]
server-side fixed by @didimitrie [here](https://github.com/speckleworks/SpeckleServer/commit/df10599f5b9ef28eb1232ea18ae96862ccec7bc5)

**Pull request closed without merging**